package.nls.json

```diff
@@ -433,9 +435,11 @@
     "DataScience.selectDifferentJupyterInterpreter": "Change Interpreter",
     "DataScience.selectJupyterInterpreter": "Select an Interpreter to start Jupyter",
     "DataScience.switchingKernelProgress": "Switching kernel to '{0}'",
-    "DataScience.sessionStartFailedWithKernel": "Failed to start a session for the Kernel '{0}'. \nView Jupyter [log](command:{1}) for further details.",
+    "DataScience.sessionStartFailedWithKernel": "Failed to start the Kernel '{0}'. \nView Jupyter [log](command:{1}) for further details.",
```

Comment:
Looks like for sentences we use kernel not Kernel in other strings in this file.